data/lib/utils.py

```
@@ -64,6 +70,31 @@ def execute_command(
 
     debug("Executing command: " + " ".join(cmd))
     debug(title)
+
+    if live_stdout:
+        lines: list[str] = []
+        process = subprocess.Popen(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)